Gemfile

```diff
@@ -2,3 +2,6 @@ source 'https://rubygems.org'
 
 gem 'fastlane', '2.222.0'
```

Comment:
Should we set the same version of Ruby for everyone?

I wonder if not setting this will cause Gemfile.lock to change due to Ruby version installed on the machine.

Reply:
Good point! I'll make a task for it.